src/ucp/api/ucp.h

```diff
@@ -169,8 +186,37 @@ enum ucp_ep_params_field {
                                                             peer */
     UCP_EP_PARAM_FIELD_ERR_HANDLING_MODE = UCS_BIT(1), /**< Error handling mode.
                                                             @ref ucp_err_handling_mode_t */
-    UCP_EP_PARAM_FIELD_ERR_HANDLER       = UCS_BIT(2)  /**< Handler to process
+    UCP_EP_PARAM_FIELD_ERR_HANDLER       = UCS_BIT(2), /**< Handler to process
                                                             transport level errors */
+    UCP_EP_PARAM_FIELD_SOCK_ADDR         = UCS_BIT(3), /**< Using a client-server
+                                                            connection establishment
+                                                            mechanism for establishing
```

Comment:
because it's not true to say "The ucp_ep_create routine will return with an error if this flag is set but the sock_address isn't specified". This (eg. UCP_EP_PARAM_FIELD_SOCK_ADDR) flag is the indicator to know if sock_address is specified or not. The flag which would cause an error if sock_addr is not specified, is UCP_EP_PARAMS_FLAGS_CLIENT_SERVER